composer.json

```diff
@@ -30,32 +30,32 @@
         "php": "^8.2",
         "ext-ctype": "*",
         "ext-iconv": "*",
-        "doctrine/doctrine-bundle": "^2.5",
-        "doctrine/doctrine-fixtures-bundle": "^3.4",
-        "friendsofsymfony/http-cache-bundle": "^2.17",
+        "doctrine/doctrine-bundle": "^2.12",
```

Comment:
we install the 2.12 recipe so we need also require it else oro extension can accidently downgrad it which is unexpected.